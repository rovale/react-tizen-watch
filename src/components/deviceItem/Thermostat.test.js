import React from 'react';
import Enzyme, { mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Thermostat from './Thermostat';

Enzyme.configure({ adapter: new Adapter() });

test('it should render as expected', () => {
  const device = {
    id: 'someDevice',
    title: 'Some Device',
    attributes: [
      { acronym: 'acronym1', value: 'value 1', unit: 'unit 1' },
    ],
  };
  const wrapper = mount((<Thermostat device={device} />)).children().first();
  expect(wrapper).toMatchSnapshot();
});
