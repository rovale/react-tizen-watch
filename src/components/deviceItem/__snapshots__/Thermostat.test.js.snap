// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render as expected 1`] = `
<div>
  Some Device
  <br />
  <div
    className="ui-li-sub-text"
  >
    acronym1
     
    value 1
    unit 1
  </div>
</div>
`;
